index-scheduler/src/lru.rs

```diff
@@ -0,0 +1,203 @@
+//! Thread-safe `Vec`-backend LRU cache using [`std::sync::atomic::AtomicU64`] for synchronization.
```

Comment:
I hope we will release the crate in the near future, just to avoid having to maintain this file in the repository :)

Reply:
Yep I'll talk about that during pre-release I think